api/forgot.go

```diff
@@ -235,6 +216,11 @@ func (a *Api) acceptPassword(res http.ResponseWriter, req *http.Request, vars ma
 
 		if usr := a.findExistingUser(rb.Email, token); usr != nil {
 
+			if !usr.HasRole("patient") {
+				a.sendModelAsResWithStatus(res, STATUS_ERR_RESET_PWD_FORBIDEN, http.StatusForbidden)
```

Comment:
same here for documentation 
// @Failure 403 {object} status.Status "Cannot send reset password to non patients users"